Header header

#3d position 
geometry_msgs/Pose pose

#buttons
byte buttonA
byte buttonB
byte buttonC

#fingers
float32[] fingers
